internal/jennies/java/templates/converters/converter.tmpl

```diff
@@ -0,0 +1,181 @@
+package {{ .Converter.Package | formatPackageName }};
+
+{{ $converter := include "converter" . }}
+{{- .Imports }}
+{{ $converter }}
+
+{{- define "guard" }}
+    {{- $leftOperand := print (.Path | formatGuardPath) -}}
+    {{- $type := .Path.Last.Type }}
+    {{- if and (eq .Op "!=") (eq .Value nil) -}}
+        {{ $leftOperand }} != null
+    {{- else if and $type.IsScalar (eq $type.AsScalar.ScalarKind "bool") -}}
+        {{- if not $type.Nullable }}
```

Comment:
This is the problem that I mentioned in Java for no `nullables`. There is only one guard in that cases and it only check the emptiness of the field when we also need the nullability.